core/iwasm/include/wasm_export.h

```diff
@@ -264,13 +264,25 @@ typedef struct LoadArgs {
 } LoadArgs;
 #endif /* LOAD_ARGS_OPTION_DEFINED */
 
+typedef struct WASMExternInstance {
+    const char *module_name;
+    const char *field_name;
+    wasm_import_export_kind_t kind;
+    union {
+        wasm_memory_inst_t memory;
+    } u;
+} WASMExternInstance;
+typedef struct WASMExternInstance *wasm_extern_inst_t;
```

Comment:
how about putting `*wasm_extern_inst_t` at L274, changing it to `} WASMExternInstance, *wasm_extern_inst_t`?